benchbuild/utils/revision_ranges.py

```diff
@@ -93,12 +93,19 @@ def __init__(self, rev_id: str, comment: tp.Optional[str] = None):
         super().__init__(comment)
         self.__id = rev_id
 
+    @property
+    def rev_id(self) -> str:
+        return self.__id
+
     def init_cache(self, repo_path: str) -> None:
         pass
 
     def __iter__(self) -> tp.Iterator[str]:
         return [self.__id].__iter__()
 
+    def __str__(self) -> str:
+        return self.__id
```

Comment:
```suggestion
        return self.rev_id
```
Using the defined property